spec/policies/comment_policy_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+describe InvoicePolicy do
+  subject { CommentPolicy }
+
+  let(:member) { FactoryGirl.build_stubbed :user }
```

Comment:
if you added `FactoryGirl::Syntax::Methods` you can use `build_stubbed` here, don't you?
Please check your PR and update the code.